lib/appSearch.js

```diff
@@ -120,12 +120,37 @@ class AppSearchClient {
     return this.client.post(`engines/${encodeURIComponent(engineName)}/documents`, documents)
   }
 
+  /**
+   * Partial update a batch of documents.
+   *
+   * @param {String} engineName unique Engine name
+   * @param {Array<Object>} documents Array of document objects to be updated.
+   * @returns {Promise<Array<Object>>} a Promise that returns an array of status objects, otherwise throws an Error.
+   */
+  updateDocuments(engineName, documents) {
+    return this.client.patch(`engines/${encodeURIComponent(engineName)}/documents`, documents)
+  }
+
+  /**
+   * List all documents
+   *
+   * @param {String} engineName unique Engine name
+   * @param {Object} options
```

Comment:
What are the supported options?

Reply:
The support options are the options that the endpoint supports: https://swiftype.com/documentation/app-search/api/documents#list

Reply:
https://github.com/elastic/app-search-node/pull/17/commits/227f19f3dcfac3055c06f305e67b175adeb4979a